Cargo.toml

```diff
@@ -37,5 +36,4 @@ lto = true
 default = ["panic-over-serial"]
 itm = []
 panic-over-itm = ["itm"]
-abort-on-panic = ["cortex-m-rt/abort-on-panic"]
```

Comment:
Is the "abort on panic" behavior removed?

If this was necessary, please explain why and note it in the Changelog.

Reply:
The `abort-on-panic` feature has been removed from `cortex-m-rt`. I believe the canonical approach is to either `extern crate panic_halt;` or to define your own (e.g., `src/panic.rs`).

I'll explain this in the changelog.